schema/test/unit/computed_columns/application_revision_naics_code_test.sql

```diff
@@ -0,0 +1,57 @@
+set client_min_messages to warning;
+create extension if not exists pgtap;
+reset client_min_messages;
+
+begin;
+select plan(2);
+
+select has_function(
+  'ggircs_portal', 'application_revision_naics_code', array['ggircs_portal.application_revision'],
+  'Function ggircs_portal.application_revision_naics_code should exist'
+);
+
+-- Init tests
+truncate ggircs_portal.organisation restart identity cascade;
+truncate ggircs_portal.naics_code restart identity cascade;
+do $$
+  declare disable_triggers json;
+  begin
+    disable_triggers := '
+      {
+        "application": ["_send_draft_application_email"],
+        "application_revision_status": ["_status_change_email"],
+        "ciip_user": ["_welcome_email"]
+      }
+    ';
+    perform test_helper.modify_triggers('disable', disable_triggers);
+  end;
+$$;
+
+select test_helper.mock_open_window();
+select test_helper.create_test_users();
+select test_helper.create_applications(1, True, True);
+update ggircs_portal.form_result set form_result = '{"operator": {"naics": "123456"}}' where application_id=1 and version_number=1 and form_id=1;
+
+insert into ggircs_portal.naics_code(naics_code, naics_description) values ('123456', 'test naics'), ('654321', 'test naics2');
+
+select * from ggircs_portal.application_revision;
+
+with record as (
+      select row(application_revision.*)::ggircs_portal.application_revision
+      from ggircs_portal.application_revision where application_id = 1 and version_number = 1
+    ) select naics_code from ggircs_portal.application_revision_naics_code((select * from record));
```

Comment:
these don't seem to be doing anything anymore?

Reply:
nope! leftover debugging :p